.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,30 @@
+## **SUBMISSIONS LACKING A RESPONSE TO ONE OF THE FIELDS WILL BE DELETED**
+
+### Issue Description:
+<!--- Use our forum https://forums.pocketmine.net for questions -->
+
+> Write a short description about the issue
+
+### Steps to reproduce the issue: 
+
+> How did you come across the issue? Did you enter a command? Did you test a plugin? Help us locate the problem by adding steps to reproduce the issue!
+
+### Operating System and Version
+<!--- Use the /version command to recieve these details. 
+PocketMine version should include the Jenkins build number or the git commit hash.
+--->
+> PocketMine-MP Version:
+PHP Version:
+Server Operating System:
+Game Version- PE or Win10: 
+
+### Crash Dump, Backtrace, or any Other Information
+
+> Please use gist or https://www.filedropper.com/ and add the link here
```

Comment:
I simply find it easier to give people the link for something as opposed to making them do the work themselves.